src/lib/transaction_database/transaction_database.ml

```diff
@@ -0,0 +1,60 @@
+open Coda_base
+open Core
+open Signature_lib
+
+(* TODO: This database should be optimized to query an individual transaction quickly. Also, it can technically take a long time to make a row update since we read a list of transactions from a user in the database and we just append the transaction into the list and then write that to the database *)
```

Comment:
Is this still true? If so should there be an associated issue?

Reply:
I will remove this